amivapi/documentation/Cheatsheet.md

```diff
@@ -22,9 +22,9 @@ other programming languages.
 ```
 /events?where={"title_en":"Party"}
 
-/events?where={"$time_start":{"$gt":"2018-06-06T10:00:00Z"}}
+/events?where={"time_start":{"$gt":"2018-06-06T10:00:00Z"}}
 
-/events?where={"$or":{"title_en":"Party","title_de":"Feier"}}
+/events?where={"$or":[{"title_en":"Party","title_de":"Feier"}]}
```

Comment:
```suggestion
/events?where={"$or":[{"title_en":"Party"},{"title_de":"Feier"}]}
```

Currently, this example does not make much sense: We have the operator `or` (fine so far), and a list of conditions (also fine). However, the list has currently only a single element, which would make the `or` quite redundant. I have added a suggestion that slightly changes it to contain two elements. Is that ok with you?

Reply:
Sure! My Bad, should have seen it myself.
Thanks!